src/pages/publication/references_tab/DegreeForm.tsx

```diff
@@ -23,38 +27,67 @@ const StyledHeading = styled.div`
   padding-top: 1.5rem;
 `;
 
-const DegreeForm: React.FC = () => {
+const DegreeForm: FC = () => {
   const { t } = useTranslation('publication');
 
   const { setFieldValue }: FormikProps<Publication> = useFormikContext();
 
+  useEffect(() => {
+    // set correct Pages type based on publication type being Degree
+    setFieldValue(ReferenceFieldNames.PAGES, null);
```

Comment:
hm, vi skulle kanskje hatt lignende for alle typene, hvor vi sikrer at all data er gyldig for valgt type... hvis vi feks endrer fra Degree til Book nå vil det kanskje bli problemer siden null ikke er gyldig PageRange (eller hva det er) som kreves av Book?

Reply:
Ja, men vi må ta en og en type av gangen. Hvis ikke blir det en helvetes svær PR.